Source/WebCore/platform/graphics/skia/GraphicsContextSkia.cpp

```diff
@@ -880,17 +880,11 @@ void GraphicsContextSkia::drawPattern(NativeImage& nativeImage, const FloatRect&
     auto samplingOptions = toSkSamplingOptions(m_state.imageInterpolationQuality());
     bool needsClip = tileRect.size() != nativeImage.size();
 
-    auto tileMode = [](float dstPoint, float dstMax, float tilePoint, float tileMax) -> SkTileMode {
-        return dstPoint >= tilePoint && dstMax <= tileMax ? SkTileMode::kClamp : SkTileMode::kRepeat;
-    };
-    auto tileModeX = tileMode(destRect.x(), destRect.maxX(), rect.x(), rect.maxX());
-    auto tileModeY = tileMode(destRect.y(), destRect.maxY(), rect.y(), rect.maxY());
-
```

Comment:
Do you know why this is wrong? I think we need an explanation in the commit message

Reply:
My experiments show that this function is called only in the cases when a pattern with repetition is being drawn. I'll add that rationale in the commit message.

Reply:
done